packages/frontend/src/components/commentBlock.tsx

```diff
@@ -72,25 +113,38 @@ const CommentBlock = ({ commentId, page }: Props) => {
                     <img src={require('../../public/images/etherscan.svg')} />
                 </a>
             </div>
-            <Link
-                className="comment-block-link"
-                to={`/post/${comment.post_id}#${comment.id}`}
-            >
+            {page === Page.Post ? (
                 <div className="block-content no-padding-horizontal">
                     <div
                         style={{
-                            maxHeight:
-                                page == Page.Home || Page.Post
-                                    ? '300px'
-                                    : undefined,
                             overflow: 'hidden',
                         }}
                         dangerouslySetInnerHTML={{
                             __html: commentHtml,
                         }}
                     />
                 </div>
-            </Link>
+            ) : (
+                <Link
```

Comment:
The `{page === Page.Post..` logic looks like it will work for now, but what are the cases where the `<Link>..` logic would run? I am not sure it is necessary.

Reply:
It's from the previous approved PR -- #169 
The `Link` will only be shown while on pages other than `Page.Post`